run/logging-manual/main.py

```diff
@@ -12,9 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START cloudrun_manual_logging]
+# [START run_manual_logging]
```

Comment:
`run_manual_logging` doesn't seem to actually be used, is this region tag needed?

Reply:
snippetbot also is complaining that the product prefix should be `cloudrun`

Reply:
Confirming the prefix should be `cloudrun` --when in doubt you an check the product in DevRel Services

Reply:
Removed that one, thanks y'all!